meson.build

```diff
@@ -0,0 +1,87 @@
+# Making releases:
+#   set the new version number:
+#     odd minor -> development series
+#     even minor -> stable series
+#     increment micro for each release within a series
+#   set nano_version to 0
+#   make the release, tag it
+#   set nano_version to 1
+
+project(
+	'telepathy-idle',
+	'c',
+	version: '0.2.0.1',
+	default_options: [
+		'c_std=c99',
+		# 'werror=@0@'.format((meson.project_version().split('.')[3] == '0').to_string()), # development version
```

Comment:
This will need to be removed: https://github.com/mesonbuild/meson/issues/4577#issuecomment-470117483